docs/blog/0.48.md

```diff
@@ -29,7 +29,7 @@ In addition you can now also invite new team members directly from the Cloud UI
 With just a click of a button you can now easily go from looking at metrics over time to bring them directly into a pivot table to further explore your metrics in a tabular format!
 
 ## Bug Fixes and Misc
-- Rebrand `share-url` changed to `public-url` in the CLI.
+- Rebrand `public-url` changed to `public-url` in the CLI.
```

Comment:
revert this ?

Reply:
Oops! Switched this one back to share-url.